### PR TITLE
[DRIP] [READY] Messenger Bags!

### PR DESCRIPTION
**Pull request by Nerev4r**
## About The Pull Request
Sprites by @zydras!
image
A new backpack option has emerged. In practice,
image
They're equivalent to a satchel, they don't have any unique mechanics.
image
Also featuring bi-color sprites for Security, and.
image

## How This Contributes To The Skyrat Roleplay Experience
More drip and more bags that look a bit more out-of-the-way is nice.

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>
image
</details>

## Changelog
:cl: Zydras
image: New messenger bags, as a loadout preference and in wardrobes/lockers!
/:cl:

### CODE REVIEW

**Comment by michealjohanson1978qqnetease** [comment]
Good

**Comment by sneksnek** [comment]
7w4700

**Comment by oniondry** [comment]
merge this NOW!!!!!!!

File: code/modules/jobs/job_types/atmospheric_technician.dm [hunk 1]
```diff
@@ -52,6 +52,7 @@
 	backpack = /obj/item/storage/backpack/industrial
 	satchel = /obj/item/storage/backpack/satchel/eng
 	duffelbag = /obj/item/storage/backpack/duffelbag/engineering
+	messenger = /obj/item/storage/backpack/messenger/eng // SKYRAT ADDITION - Messenger Bags
```
Comment (softcerv) [inline code/modules/jobs/job_types/atmospheric_technician.dm]: you can make this and the other changes like this modular

File: code/modules/mining/mine_items.dm [hunk 2]
```diff
@@ -36,6 +36,7 @@
 	new /obj/item/storage/backpack/duffelbag/explorer(src)
 	new /obj/item/storage/backpack/explorer(src)
 	new /obj/item/storage/backpack/satchel/explorer(src)
+	new /obj/item/storage/backpack/messenger/explorer(src) //SKYRAT ADDITION - Messenger Bags
```
Comment (softcerv) [inline code/modules/mining/mine_items.dm:39]: I think we have a modular vendor variable that we can use for this

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 3]
```diff
@@ -8,7 +8,8 @@
 		/obj/item/clothing/under/rank/medical/scrubs/skyrat/red = 4,
 		/obj/item/clothing/under/rank/medical/scrubs/skyrat/white = 4,
 		/obj/item/clothing/under/rank/medical/doctor/skyrat/utility = 4,
-		/obj/item/clothing/suit/toggle/labcoat/medical = 4
+		/obj/item/clothing/suit/toggle/labcoat/medical = 4,
+		/obj/item/storage/backpack/messenger/med = 4
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/med = 4,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 4]
```diff
@@ -23,13 +24,15 @@
 		/obj/item/clothing/head/utility/hardhat/welding/orange = 2,
 		/obj/item/clothing/head/utility/hardhat/dblue = 2,
 		/obj/item/clothing/head/utility/hardhat/welding/dblue = 2,
-		/obj/item/clothing/head/utility/hardhat/red = 2
+		/obj/item/clothing/head/utility/hardhat/red = 2,
+		/obj/item/storage/backpack/messenger/eng = 3
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/eng = 3,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 5]
```diff
@@ -23,13 +24,15 @@
...
 	)
 
 /obj/machinery/vending/wardrobe/atmos_wardrobe
 	skyrat_products = list(
 		/obj/item/clothing/glasses/meson/engine = 2,
 		/obj/item/clothing/head/beret/atmos = 4,
+		/obj/item/storage/backpack/messenger/eng = 3
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/eng = 3,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 6]
```diff
@@ -43,6 +46,7 @@
 		/obj/item/clothing/suit/gorka/supply = 3,
 		/obj/item/clothing/suit/toggle/jacket/supply = 3,
 		/obj/item/clothing/glasses/hud/gun_permit = 5, //from company imports module
+		/obj/item/storage/backpack/messenger = 3
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger = 3,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 7]
```diff
@@ -61,20 +65,23 @@
 		/obj/item/storage/backpack/science/robo = 2,
 		/obj/item/storage/backpack/satchel/science/robo = 2,
 		/obj/item/storage/backpack/duffelbag/science/robo = 2,
+		/obj/item/storage/backpack/messenger/science/robo = 2
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/science/robo = 2,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 8]
```diff
@@ -61,20 +65,23 @@
...
 	)
 
 /obj/machinery/vending/wardrobe/science_wardrobe
 	skyrat_products = list(
 		/obj/item/clothing/under/rank/rnd/scientist/skyrat/hlscience = 3,
 		/obj/item/clothing/under/rank/rnd/scientist/skyrat/utility = 3,
 		/obj/item/clothing/suit/toggle/jacket/sci = 3,
+		/obj/item/storage/backpack/messenger/science = 3
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/science = 3,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 9]
```diff
@@ -61,20 +65,23 @@
...
 	)
 
 /obj/machinery/vending/wardrobe/hydro_wardrobe
 	skyrat_contraband = list(
 		/obj/item/clothing/under/suit/skyrat/scarface = 2,
 		/obj/item/clothing/under/misc/skyrat/gear_harness/adam = 2,
 		/obj/item/clothing/under/misc/skyrat/gear_harness/eve = 2,
+		/obj/item/storage/backpack/messenger/hyd = 3
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/hyd = 3,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 10]
```diff
@@ -110,11 +117,13 @@
 		/obj/item/clothing/under/rank/medical/chemist/skyrat/formal = 2,
 		/obj/item/clothing/under/rank/medical/chemist/skyrat/formal/skirt = 2,
 		/obj/item/clothing/head/beret/medical/chemist = 2,
+		/obj/item/storage/backpack/messenger/chem = 2
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/chem = 2,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 11]
```diff
@@ -110,11 +117,13 @@
...
 	)
 
 /obj/machinery/vending/wardrobe/viro_wardrobe
 	skyrat_products = list(
 		/obj/item/clothing/head/beret/medical/virologist = 2,
+		/obj/item/storage/backpack/messenger/vir = 2
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/vir = 2,
```

File: modular_skyrat/modules/modular_vending/code/wardrobes.dm [hunk 12]
```diff
@@ -130,6 +139,7 @@
 		/obj/item/clothing/suit/toggle/labcoat/skyrat/highvis = 2,
 		/obj/item/clothing/under/rank/security/detective/cowboy = 2,
 		/obj/item/clothing/suit/jacket/det_suit/cowboyvest = 2,
+		/obj/item/storage/backpack/messenger/sec = 2
```
Comment (softcerv) [inline modular_skyrat/modules/modular_vending/code/wardrobes.dm]:
```suggestion
		/obj/item/storage/backpack/messenger/sec = 2,
```

File: code/modules/jobs/job_types/chemist.dm [hunk 13]
```diff
@@ -54,7 +54,8 @@
 
 	backpack = /obj/item/storage/backpack/chemistry
 	satchel = /obj/item/storage/backpack/satchel/chem
-	duffelbag = /obj/item/storage/backpack/duffelbag/chemistry
+	duffelbag = /obj/item/storage/backpack/duffelbag/
```
Comment (softcerv) [inline code/modules/jobs/job_types/chemist.dm]:
```suggestion
	duffelbag = /obj/item/storage/backpack/duffelbag/chemistry
```

File: code/modules/jobs/job_types/chief_engineer.dm [hunk 14]
```diff
@@ -71,7 +71,8 @@
 
 	backpack = /obj/item/storage/backpack/industrial
 	satchel = /obj/item/storage/backpack/satchel/eng
-	duffelbag = /obj/item/storage/backpack/duffelbag/engineering
+	duffelbag = /obj/item/storage/backpack/duffelbag/
```
Comment (softcerv) [inline code/modules/jobs/job_types/chief_engineer.dm]:
```suggestion
	duffelbag = /obj/item/storage/backpack/duffelbag/engineering
```

File: modular_skyrat/master_files/code/game/objects/items/storage/backpack.dm [hunk 15]
```diff
@@ -36,3 +36,121 @@
...
+	name = "security messenger bag"
+	desc = "A robust messenger bag for security related needs."
+	icon_state = "messenger_security_black"
+	inhand_icon_state = "messenger_security_black"
+	uses_advanced_reskins = TRUE
+	unique_reskin = list(
+		"Black Variant" = list(
+			RESKIN_ICON_STATE = "messenger_security_black",
+			RESKIN_WORN_ICON_STATE = "messenger_security_black",
+			RESKIN_INHAND_STATE = "messenger_security_black"
```
Comment (softcerv) [inline modular_skyrat/master_files/code/game/objects/items/storage/backpack.dm]:
```suggestion
			RESKIN_INHAND_STATE = "messenger_security_black",
```

File: modular_skyrat/master_files/code/game/objects/items/storage/backpack.dm [hunk 16]
```diff
@@ -36,3 +36,121 @@
...
+		),
+		"White Variant" = list(
+			RESKIN_ICON_STATE = "messenger_security_white",
+			RESKIN_WORN_ICON_STATE = "messenger_security_white",
+			RESKIN_INHAND_STATE = "messenger_security_white"
```
Comment (softcerv) [inline modular_skyrat/master_files/code/game/objects/items/storage/backpack.dm]:
```suggestion
			RESKIN_INHAND_STATE = "messenger_security_white",
```

**Review by softcerv — COMMENTED** [review]
Here are some things I noticed

**Comment by YakumoChen** [comment]
This seems like a perfectly good change for upstream, minus the lack of actual security colours needed

**Pull request closed without merging**